File: typical90/Swift/014.swift
func main() {
    let n = Int(readLine()!)!
    let a = readLine()!.split(separator: " ").map{Int($0)!}.sorted()
    let b = readLine()!.split(separator: " ").map{Int($0)!}.sorted()

    var ans = 0;
    for i in 0 ..< n {
        ans += abs(a[i]-b[i]);
    }
    print(ans)
}
main()

File: typical90/Python/014.py
def main():
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))
    
    a.sort()
    b.sort()
    
    ans = 0
    for x, y in zip(a,b):
        ans += abs(x-y)
    print(ans)

if __name__ == '__main__':
    main()